unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEmotesSectionController.cs

```diff
@@ -135,9 +142,67 @@ private void EmoteSell(string collectibleId)
             var ownedCollectible = ownedNftCollectionsL1.FirstOrDefault(nft => nft.urn == collectibleId) ??
                                    ownedNftCollectionsL2.FirstOrDefault(nft => nft.urn == collectibleId);
 
-            WebInterface.OpenURL(ownedCollectible != null ?
-                URL_SELL_SPECIFIC_COLLECTIBLE.Replace("{collectionId}", ownedCollectible.collectionId).Replace("{tokenId}", ownedCollectible.tokenId) :
-                URL_SELL_COLLECTIBLE_GENERIC);
+            WebInterface.OpenURL(ownedCollectible != null ? URL_SELL_SPECIFIC_COLLECTIBLE.Replace("{collectionId}", ownedCollectible.collectionId).Replace("{tokenId}", ownedCollectible.tokenId) : URL_SELL_COLLECTIBLE_GENERIC);
+        }
+
+        private async UniTask<IEnumerable<WearableItem>> FetchCustomEmoteItems(CancellationToken cancellationToken)
+        {
+            IReadOnlyList<string> customItems = await customNftCollectionService.GetConfiguredCustomNftItemsAsync(cancellationToken);
+
+            IEnumerable<string> publishedItems = customItems
```

Comment:
You can merge two loops together and just have 2 branches there. Moreover, I believe you can process them in parallel. The current way of doing things leads to unnecessary allocations and repetitive pieces of logic. 
Unless you really require to fetch `publishedItems` first but, hopefully, there is no such dependency

Reply:
Done